File: gui-tjenester/src/main/java/no/nav/melosys/tjenester/gui/dto/PeriodeDto.java
package no.nav.melosys.tjenester.gui.dto;

import java.time.LocalDate;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonPropertyOrder;

@JsonPropertyOrder({
        "fom",
        "tom"
})
public class PeriodeDto {

    @JsonProperty("fom")
    private LocalDate fom;
    @JsonProperty("tom")
    private LocalDate tom;
    
    public PeriodeDto() {
    }

    /**
     * @param tom
     * @param fom
     */
    public PeriodeDto(LocalDate fom, LocalDate tom) {
        super();
        this.fom = fom;
        this.tom = tom;
    }

    public LocalDate getFom() {
        return fom;
    }

    public void setFom(LocalDate fom) {
        this.fom = fom;
    }

    public LocalDate getTom() {
        return tom;
    }

    public void setTom(LocalDate tom) {
        this.tom = tom;
    }
}

File: service/src/main/kotlin/no/nav/melosys/service/oppgave/dto/PeriodeDto.kt
package no.nav.melosys.service.oppgave.dto

import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.annotation.JsonPropertyOrder
import java.time.LocalDate

@JsonPropertyOrder("fom", "tom")
data class PeriodeDto(
    @JsonProperty("fom")
    val fom: LocalDate?,

    @JsonProperty("tom")
    val tom: LocalDate?
)
